### PR TITLE
This adds support for standards installation through Composer.

### PR DESCRIPTION
**Pull request by schlessera**
This depends on the following pull request for the `squizlabs/php_codesniffer` package, and should only be merged once that pull request has been merged as well:
https://github.com/squizlabs/PHP_CodeSniffer/pull/936

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,6 +1,8 @@
 {
 	"name"       : "wp-coding-standards/wpcs",
 	"description": "PHP_CodeSniffer rules (sniffs) to enforce WordPress coding conventions",
+	"type"       : "phpcs-standard",
+	"version": "9.0",
```
Comment (GaryJones) [inline composer.json]: Where does this version 9.0 come from?

Comment (schlessera) [inline composer.json]: TBH, I have no idea. Was certainly not my intention to introduce that one here (and the formatting is off as well).
I'll adapt the pull request...

**Comment by Rarst** [comment]
I think by now #855 (using third party installer plugin) is more realistic way to go since upstream PR doesn't seem to be happening.

**Comment by jrfnl** [comment]
@schlessera I believe this PR was superseded by https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/pull/855 ? Do you agree or is there still something we're missing ?

**Comment by schlessera** [comment]
@jrfnl I agree and will close this one.

**Pull request closed without merging**